cypress/support/utils.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+function createUUID() {
+    var s = [];
+    var hexDigits = "0123456789abcdef";
+    for (var i = 0; i < 36; i++) {
+        s[i] = hexDigits.substr(Math.floor(Math.random() * 0x10), 1);
+    }
+    s[14] = "4";
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    s[19] = hexDigits.substr(s[19] | 0x8, 1);
+    s[8] = s[13] = s[18] = s[23] = "-";
+    var uuid = s.join("");
+    return uuid;
+}
+exports.createUUID = createUUID;
+function getAuthCodeFromLocation(location) {
```

Comment:
This might also be something a URL lib could help simplify

Reply:
this function was obselete so I removed it!